doc/refman/RefMan-pre.tex

```diff
@@ -1087,6 +1087,135 @@ \section*{Credits: version 8.5}
 Hugo Herbelin, Matthieu Sozeau and the {\Coq} development team\\
 \end{flushright}
 
+\section*{Credits: version 8.6 (Stronger, Better, Faster Rooster)}
+
+{\Coq} version 8.6 contains the result of refinements, stabilization of
+8.5's features and cleanups of the internals of the system. Over the
+year of (now time-based) development, about 450 bugs were resolved and
+over 100 contributions integrated. The main user visible changes are:
+\begin{itemize}
+\item A new, state-of-the-art universe constraint checker that
+  can outperform the previous version by an order of magnitude, by
+  Jacques-Henri Jourdan.
+\item In CoqIDE and other asynchronous interfaces, more fine-grained
+  asynchronous processing and error reporting by Enrico Tassi.
+\item More access to the proof engine features from Ltac: goal
+  management primitives, range selectors and a {\tt typeclasses
+    eauto} engine handling multiple goals and multiple successes, by
+  Cyprien Mangin, Matthieu Sozeau and Arnaud Spiwack.
+\item Tactic behavior uniformization and specification, generalization
+  of intro-patterns by Hugo Herbelin and others.
+\item Update of the beautifier by Hugo Herbelin, useful for switching
+  between versions.
+\item A brand new warning system allowing to control warnings, turn them
+  into errors or ignore them selectively by Maxime Dénès, Guillaume
+  Melquiond and others.
+\item Irrefutable patterns in abstractions, by Daniel de Rauglaudre.
+\item Integration of {\tt ssreflect}'s subterm selection algorithm by
+  Enrico Tassi.
+\item Integration of {\tt LtacProf}, a profiler for {\tt Ltac} by Tobias
+  Tebbi, Jason Gross and Paul Steckler.
```

Comment:
In the integration process, I rewrote 80% of it to make it STM compatible (1 week of work, damn)...  I think you should put my name here.